app/src/app/routes/Sources/Mappings/MappingHeader.tsx

```diff
@@ -1,27 +1,27 @@
 import React, { useState } from "react";
 
+import { Icon } from "@blueprintjs/core";
+import { IconNames } from "@blueprintjs/icons";
 import {
-  Button,
   Divider,
   IconButton,
   makeStyles,
   Menu,
   MenuItem,
   ListItemText,
-  Typography,
 } from "@material-ui/core";
 import MoreIcon from "@material-ui/icons/MoreHoriz";
-import PlayIcon from "@material-ui/icons/PlayArrow";
 import { useTranslation } from "react-i18next";
 import { useHistory, useLocation, useParams } from "react-router-dom";
 
+import Button from "common/components/Button";
 import MappingDeleteDialog from "features/Mappings/Delete/MappingDeleteDialog";
 import MappingNameDialog from "features/Mappings/Edit/MappingNameDialog";
 import NavigationBreadcrumbs from "features/NavigationBreadcrumbs/NavigationBreadcrumbs";
 
 const useStyles = makeStyles((theme) => ({
   root: {
-    paddingInline: theme.spacing(5),
+    padding: theme.spacing(0, 5, 0, 5),
```

Comment:
Correct me if i'm wrong, but you should be able to do so :

```suggestion
    padding: theme.spacing(0, 5),
```

Reply:
indeed !